docs/sources/guides/whats-new-in-v6-0.md

```diff
@@ -113,30 +114,39 @@ will be shared closer to or just after release.
 
 The Azure Monitor datasource integrates four Azure services with Grafana - Azure Monitor, Azure Log Analytics, Azure Application Insights and Azure Application Insights Analytics.
 
-### Provisioning support for alert notifiers
+## Provisioning support for alert notifiers
 
 Grafana now added support for provisioning alert notifiers from configuration files. Allowing operators to provision notifiers without using the UI or the API. A new field called `uid` has been introduced which is a string identifier that the administrator can set themselves. Same kind of identifier used for dashboards since v5.0. This feature makes it possible to use the same notifier configuration in multiple environments and refer to notifiers in dashboard json by a string identifier instead of the numeric id which depends on insert order and how many notifiers that exists in the instance.
 
-### Auth and session token improvements
+## Easier to deploy & more secure
 
-The previous session storage implementation in Grafana was causing problems in larger HA setups due to too many write requests to the database. The remember me token also have several security issues which is why we decided to rewrite auth middleware in Grafana and remove the session storage since most operations using the session storage could be rewritten to use cookies or data already made available earlier in the request.
-If you are using `Auth proxy` for authentication the session storage will still be used but our goal is to remove this ASAP as well.
+Grafana 6.0 removes the need of configuring and setup of additional storage for [user sessions](/tutorials/ha_setup/#user-sessions). This should make it easier to deploy and operate Grafana in a
+high availability setup and/or if you're using a stateless user session storage like Redis, Memcache, Postgres or MySQL.
 
-This release will force all users to log in again since their previous token is not valid anymore.
+Instead of user sessions a solution based on short-lived tokens that are rotated frequently have been implemented. This also replaces the old "remember me cookie"
+solution, which allowed a user to be logged in between browser sessions, and which have been subject to several security holes throughout the years.
+Read more about the short-lived token solution and how to configure it [here](/auth/overview/#login-and-short-lived-tokens).
 
-### Named Colors
+> Please note that due to these changes, all users will be required to login upon next visit after upgrade.
+
+Besides these changes there have also been security improvements regarding protection against Cross-Site Request Forgery (CSRF) and Cross-site Scripting (XSS) attacks.
```

Comment:
I think we could mention SameSite and link to owasp docs.

Reply:
Yeah thought of that but felt like to much technical details in this post. Thought that we could maybe link to other page(s) given that we write up a security best practices or similar guide.